lib/models/finamp_models.dart

```diff
@@ -215,6 +217,9 @@ class FinampSettings {
   bool hideSongArtistsIfSameAsAlbumArtists =
       _hideSongArtistsIfSameAsAlbumArtists;
 
+  @HiveField(47, defaultValue: _showArtistsTopSongs)
+  bool showArtistsTopSongs = _showArtistsTopSongs;
+
```

Comment:
Please move this to the end, so it's easier to figure out what the next ID should be